From 946c13ecd0d37fe0f863be7892151e9b4264c013 Mon Sep 17 00:00:00 2001
From: Kirill Kapranov <kirill.kapranov@compulab.co.il>
Date: Mon, 25 Oct 2021 23:39:29 +0300
Subject: [PATCH 46/84] defconfig: Fix PCIe and mPCIE_USB

Regression: these interfaces stop working because the driver reset_imx7 has
been excluded after some Kconfigs cleaning. Thus a card in mPCIe slot left
uninitialized, even being bare USB slave

Signed-off-by: Kirill Kapranov <kirill.kapranov@compulab.co.il>
---
 arch/arm64/configs/cl-imx8m-mini_defconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/configs/cl-imx8m-mini_defconfig b/arch/arm64/configs/cl-imx8m-mini_defconfig
index 0182fa452715..a764b4022657 100644
--- a/arch/arm64/configs/cl-imx8m-mini_defconfig
+++ b/arch/arm64/configs/cl-imx8m-mini_defconfig
@@ -813,3 +813,4 @@ CONFIG_MEMTEST=y
 CONFIG_CORESIGHT=y
 CONFIG_CORESIGHT_LINK_AND_SINK_TMC=y
 CONFIG_CORESIGHT_SOURCE_ETM4X=y
+CONFIG_RESET_IMX7=y
-- 
2.11.0

